### ISSUE TITLE
Packages directory filenames are too long to be archived via Phar

### ISSUE THREAD

**Issue by Stoyan0v** [woocommerce/woocommerce#26691]
**Describe the bug**
I'm not sure if this is the right type of issue I'm going to report, so please excuse me if I haven't picked the right one.

I'm one of the developers of [Siteground Migrator](https://wordpress.org/plugins/siteground-migrator/) plugin, and we have issues migrate WooCommerce sites after 4.0.

The problem is that the filenames generated in packages folder are too long, and we cannot create tar archive when we migrate sites to our servers.
Here is an example: `woocommerce/packages/woocommerce-admin/dist/chunks/vendors~analytics-report-categories~analytics-report-coupons~analytics-report-downloads~analytics-re~2579715d.3c56dd5478f01faa4b26.js`

We are using [Phar::buildFromDirectory](https://www.php.net/manual/en/phar.buildfromdirectory.php), but we get the following error when we try to archive the woocommerce directory:
`Uncaught PharException: dark-based phar packages / woocommerce-admin / dist / chunks / vendors ~ analytics-report-categories ~ analytics-report-coupons ~ analytics-report-downloads ~ analytics-re ~ 2579715d.3c56dd5478f01faa4b26.js "is too long for tar file format`

Is there any chance to change how filenames are generate, so they can become shorter?

**Comment by STV11C** [comment]
Yes agree, similar problem on my end for 6 files where I cannot commit to my projects Git repo. Screen attached. Thanks.

<img width="987" alt="WC-capture" src="https://user-images.githubusercontent.com/5532744/83835280-ba5c4000-a733-11ea-92a9-da7e418e6f78.PNG">

**Comment by juliaamosova** [comment]
Hi @Stoyan0v,

Thank you for reporting the issue. It seems to be on the WooCommerce Admin side of things which is being developed in another repository: https://github.com/woocommerce/woocommerce-admin

I am going to move your bug report there so that the right team could take a look further. Thanks for your patience!
